makefile

```diff
@@ -236,6 +236,15 @@ run-tests/%: tests/%.dx just-build
 run-examples/%: examples/%.dx just-build
 	misc/check-quine $< $(dex) script
 
+lower-tests: export DEX_LOWER=1
+lower-tests: export DEX_ALLOW_CONTRACTIONS=0
+lower-tests: export DEX_TEST_MODE=t
+lower-tests: just-build
+	misc/check-quine tests/lower.dx $(dex) script
+update-lower-tests: just-build
```

Comment:
Should we set `DEX_LOWER=1` etc for the `update` target as well?

Reply:
I guess it doesn't matter for the update target, since they should both produce the same results. So it might be an even better test to update using the golden implementation without this pass, but check that the results don't change once we enable it!

But it was an accident, good catch.